.github/workflows/publish-maven-snapshots.yml

```diff
@@ -5,8 +5,8 @@ on:
   push:
       branches:
         - main
-        - '1.3'
-        - 2.x
+        - '[0-9]+.[0-9]+'
```

Comment:
I think it won't hurt: we usually don't backport anything to older branches, so we shouldn't be flooding the repository with unneeded snapshots